internal/job/git.go

```diff
@@ -34,14 +34,21 @@ const (
 	gitErrorCleanSubmodules
 )
 
+const (
+	gitErrStrbadObject             = "fatal: bad object"
```

Comment:
Capital B for bad in this error string?

Reply:
Yeah I don't think so - this is for matching the string output by git.